scripts/use-env.sh

```
@@ -0,0 +1,25 @@
+# active environment with credentials that work there. Never edit .env in
+# place; edit the profile and re-run this script.
+#
+# Usage: scripts/use-env.sh <profile>     e.g. scripts/use-env.sh local
+set -euo pipefail
+cd "$(dirname "$0")/.."
+
+profile="${1:-}"
+if [ -z "$profile" ] || [ ! -f ".env.$profile" ]; then
+  echo "usage: scripts/use-env.sh <profile>   (needs client/.env.<profile>)"
+  echo "available profiles:"
+  ls -1 .env.* 2>/dev/null | grep -v example | sed 's/^\.env\./  /' || echo "  (none — create .env.local / .env.staging first)"
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)